tools/update-rust.sh

```diff
@@ -14,7 +14,7 @@ fi
 
 (
     cd rust
-    current_branch=`git branch --show-current`
+    current_branch=`git name-rev --name-only HEAD`
```

Comment:
Is this related to the BV changes?  If not, perhaps it should be handled separately?

Reply:
Ah yes, this is not related to the bv change. I will remove this from the pull request. I have been running these on the server, but I don't have the privilege to upgrade the git version. The old git does not recognize the command there.